### PR TITLE
Loading screen

### PR DESCRIPTION
**Pull request by mdsams**

### CODE REVIEW

**Comment by cosmo3769** [comment]
Hey @mdsams Looks real cool. One point to look over: 

* The loading screen is not loading completely and the next screen which in your case is speaker is showing.

File: src/App.tsx [hunk 1]
```diff
@@ -1,11 +1,13 @@
-import React from "react";
+import React from 'react';
+import Speakers from './Components/Speakers/Speakers';
 
 function App() {
```
Comment (cosmo3769) [inline src/App.tsx:19]: We can use {useState, useEffect} in the App function to have a loading timer so to render it completely. 

```
function App() {

  const [loading, setLoading] = useState(true);
  useEffect(() => {
    setTimeout(() => {
      setLoading(false);
    }, 3500);
  }, []);

    return (
        <div>
            {/* <h1 className='text-3xl font-bold underline'>Cloud Community Days</h1> */}
            {loading ? 
            <Loading/>
            :
            <Speakers />}  
        </div>
    );
}
```

File: src/index.tsx [hunk 2]
```diff
@@ -1,13 +1,18 @@
...
+import ReactDOM from 'react-dom/client';
+import './index.css';
+import App from './App';
+import Loading from './Components/Loading';
 
-const root = ReactDOM.createRoot(
-  document.getElementById("root") as HTMLElement
-);
+const root = ReactDOM.createRoot(document.getElementById('root') as HTMLElement);
+const footer = ReactDOM.createRoot(document.getElementById('footer') as HTMLElement);
```
Comment (cosmo3769) [inline src/index.tsx]: After making changes in App.tsx file, we can remove this code line. (const footer)

File: src/index.tsx [hunk 3]
```diff
@@ -1,13 +1,18 @@
...
 root.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>
+    <React.StrictMode>
+        <App />
+    </React.StrictMode>,
+);
+footer.render(
```
Comment (cosmo3769) [inline src/index.tsx]: We can remove this `footer.render`.

**Review by cosmo3769 — CHANGES_REQUESTED** [review]

File: src/App.tsx (hunk 1 above)
Comment (mdsams) [reply src/App.tsx:19]: Thanks cosmos

**Review by cosmo3769 — APPROVED** [review]

**Review by avinashkranjan — APPROVED** [review]
LGTM..✅

**Pull request merged**